.github/workflows/perf.yaml

```
@@ -0,0 +1,40 @@
+name: Akka Strem Perf Spec
+
+on:
+  schedule:
+    - cron: "0 0 * * *"
+
+jobs:
+  test:
+    name: Test
+    runs-on: self-hosted
+    if: github.repository == 'akka/akka'
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Cache Coursier cache
+        uses: coursier/cache-action@v6.2
+
+      - name: Run Performance tests
+        run: |
+          cp .jvmopts-perf-ci .jvmopts
+          sbt \
+            -Dakka.ci-server=true \
+            -Dakka.test.timefactor=1 \
+            -Djava.security.egd=file:/dev/./urandom \
+            -Dakka.test.timefactor=1 \
+            "akka-bench-jmh/jmh:compile" \
+            "akka-bench-jmh/jmh:runMain akka.BenchRunner full -rf csv -rff GraphBuilderBenchmark.csv -p complexity=10 akka.stream.GraphBuilderBenchmark" \
+            "akka-bench-jmh/jmh:runMain akka.BenchRunner full -rf csv -rff MaterializationBenchmark.csv -p complexity=10 akka.stream.MaterializationBenchmark"
```

```suggestion
            "akka-bench-jmh/Jmh/compile" \
            "akka-bench-jmh/Jmh/runMain akka.BenchRunner full -rf csv -rff GraphBuilderBenchmark.csv -p complexity=10 akka.stream.GraphBuilderBenchmark" \
            "akka-bench-jmh/Jmh/runMain akka.BenchRunner full -rf csv -rff MaterializationBenchmark.csv -p complexity=10 akka.stream.MaterializationBenchmark"
```

Just to minimize the number of deprecated sbt features we use.

---

ok